search/search_params.hpp

```diff
@@ -20,6 +20,7 @@ class Tracer;
 
 struct SearchParams
 {
+  static size_t const kDefaultNumBookmarksResults = 1000;
```

Comment:
s/BookmarksResults/ResultsBookmarks/ for consistency

Reply:
But what is the right naming? From my point of view the current naming is better.